modules/comment/comment.resolver.js

```diff
@@ -0,0 +1,15 @@
+const commentsService = require('./comment.service');
+
+const commentsQuery = {
+  getAllComments: () => commentsService.getAllComments(),
+  getCommentById: (parent, args) => commentsService.getCommentById(args.id),
```

Comment:
Please, check if the document is found and throw an error if the comment is not found. Look at the news resolver and its error handling.

Reply:
Error handling will be in the next pull request